buildSrc/src/main/kotlin/Releases.kt

```diff
@@ -58,12 +76,37 @@ object Releases {
     const val versionCode = 1
     const val versionName = "1.0"
   }
+}
+
+fun Project.publishAsArtifact(artifact: LibraryArtifact) {
```

Comment:
nit: somehow this reads better? optional.

```suggestion
fun Project.publishArtifact(artifact: LibraryArtifact) {
```